src/common/html-attributes/index.js

```diff
@@ -12,21 +12,24 @@ function camelToKebab(s) {
  * Create object of HTML attributes for pass-through to the DOM
  * @param {Object} input
  */
-function processHtmlAttributes(input = {}) {
+function processHtmlAttributes(input = {}, ignore) {
     const attributes = {};
     const htmlAttributes = input.htmlAttributes;
-    const wildcardAttributes = input['*'];
 
-    let obj = htmlAttributes || wildcardAttributes;
-    if (htmlAttributes && wildcardAttributes) {
-        obj = assign(wildcardAttributes, htmlAttributes);
+    let obj = htmlAttributes || {};
+    if (htmlAttributes) {
+        obj = assign({}, htmlAttributes);
     }
-
-    if (obj) {
-        Object.keys(obj).forEach((key) => {
-            attributes[camelToKebab(key)] = obj[key];
+    if (!!ignore) {
```

Comment:
Is there a case where there isn't an ignore?

Reply:
That was left in order to validate tests so I could check if it was using the old syntax. I can remove it